src/seir.py

```diff
@@ -2,12 +2,10 @@
 
 
 def main():
-    # Open and run a 300-day simulation using the SEIR model.
+    # Open and run the SEIR simulation.
 
     simulation = oc.open_simulation(
-        'https://models.physiomeproject.org/workspace/5d4/rawfile/b973009caf430b1636cfd1133a23bfd570a85691/COVIDSim.cellml')
-
-    simulation.data().set_ending_point(300)
```

Comment:
not a big deal, but I would generally still set the ending point in the python code.

Reply:
With this script, I really only wanted to mimic @ssafaei's SED-ML file. Eventually, the script will support arguments, one of which will no doubt be the duration of the simulation.